File: Python projects/Python Basics/conditional_statements/exercises/shopping.py
budget = float(input())
gpus = int(input())
cpus = int(input())
ram = int(input())
cost_gpus = gpus * 250
cost_cpus = cpus * cost_gpus * 0.35
cost_ram = ram * cost_gpus * 0.1
total_cost = cost_gpus + cost_cpus + cost_ram
if gpus > cpus:
    total_cost = total_cost * 0.85

if budget >= total_cost:
    print(f"You have {budget - total_cost:.2f} leva left!")
else:
    print(f"Not enough money! You need {total_cost - budget:.2f} leva more!")

File: JavaScript projects/JavaScript Basics/Conditional Statements/exercises/shopping.js
function shopping(budget, countGpus, countCpus, countRam){
    let priceGpus = countGpus * 250;
    let priceCpus = priceGpus * 0.35 * countCpus;
    let priceRam = priceGpus * 0.1 * countRam;
    let totalPrice = priceGpus + priceCpus + priceRam;

    if (countCpus < countGpus)
        totalPrice *= 0.85;

    if (totalPrice <= budget){
        console.log(`You have ${(budget - totalPrice).toFixed(2)} leva left!`);
    }else{
        console.log(`Not enough money! You need ${(totalPrice - budget).toFixed(2)} leva more!`);
    }
}

shopping(920.45, 3, 1, 1);